t/t2403-job-exec-conf.t

```diff
@@ -100,5 +100,79 @@ test_expect_success 'job-exec: bad imp config causes module failure' '
 	) &&
 	grep "error reading config value exec.imp" ${name}.log
 '
-
+test_expect_success 'job-exec: can specify exec service on cmdline' '
+	flux dmesg -C &&
+	flux module reload -f job-exec service=foo &&
+	flux dmesg &&
+	flux dmesg | grep "using exec service foo"
+'
+test_expect_success 'job-exec: exec service can be set in exec conf' '
+	name=exec-service &&
+	mkdir ${name}.d &&
+	cat <<-EOF > ${name}.d/exec.toml &&
+	[exec]
+	service = "bar"
+	EOF
+	( export FLUX_CONF_DIR=${name}.d &&
+	  flux start -s1 flux dmesg > ${name}.log 2>&1
+	) &&
+	grep "using exec service bar" ${name}.log
```

Comment:
This could be simplfiied by using the broker `--config-path` option which can take a single toml file:
```
        cat <<-EOF > ${name}.toml &&
	[exec]
	service = "bar"
	EOF
	flux start -o,--config-path=${name}.toml -s1 flux dmesg >${name}.log 2>&1 &&
	grep "using exec service bar" ${name}.log
```
Same for the tests that follow

Reply:
I was copying the style that was in the file already with other tests.  I'll update all the tests.